src/components/Api/Api.jsx

```diff
@@ -0,0 +1,295 @@
+                        <StripeCheckout
+                          name="OpenDota"
+                          description={strings.api_title}
+                          billingAddress
+                          stripeKey={process.env.REACT_APP_STRIPE_PUBLIC_KEY}
+                          token={this.handleUpdate}
+                          zipCode
+                          locale="auto"
+                        >
+                          <RaisedButton label={strings.api_update_billing} style={{ margin: '5px 5px' }} />
+                        </StripeCheckout>
+                      </div>
+                      : <div />
+                    }
+                    {
+                      this.state.usage ?
+                        <div>
+                          <h4>{strings.api_header_usage}</h4>
+                          <Table style={{ width: '60%', margin: '0 auto' }}>
+                            <TableHeader
+                              displaySelectAll={false}
+                              adjustForCheckbox={false}
+                            >
+                              <TableRow>
+                                <TableHeaderColumn>{strings.api_month}</TableHeaderColumn>
+                                <TableHeaderColumn>{strings.api_usage_calls}</TableHeaderColumn>
+                              </TableRow>
+                            </TableHeader>
+                            <TableBody
+                              displayRowCheckbox={false}
+                            >
+                              { this.state.usage.map(e => (
+                                <TableRow key={e.month}>
+                                  <TableRowColumn>{e.month}</TableRowColumn>
+                                  <TableRowColumn>{e.usage_count}</TableRowColumn>
+                                </TableRow>))}
+                            </TableBody>
+                          </Table>
+                        </div>
+                      : <div />
+                    }
+                  </div>
+                  : <div />
+                }
+              </div>
+          }
+          <h3>{strings.api_header_price}</h3>
+          <Table style={{ width: '60%', margin: '0 auto' }}>
+            <TableHeader
+              displaySelectAll={false}
+              adjustForCheckbox={false}
+            >
+              <TableRow>
+                <TableHeaderColumn>{`${strings.api_first} 25000 ${strings.api_calls_month}`}</TableHeaderColumn>
+                <TableHeaderColumn>{`>25000 ${strings.api_calls_month}`}</TableHeaderColumn>
+              </TableRow>
+            </TableHeader>
+            <TableBody
+              displayRowCheckbox={false}
+            >
+              <TableRow>
+                <TableRowColumn>{strings.api_free}</TableRowColumn>
+                <TableRowColumn>{`$1.00 ${strings.api_per_unit}`}</TableRowColumn>
+              </TableRow>
```

Comment:
I would also add to the table the increased rate limit per minute, and offer priority support for customers with keys.

Reply:
I think moving more of the info into the table would also help reduce the "wall of text" on the details

Reply:
Suggest making this a bit more colorful, maybe add icons (check and X, etc.)?